frontend/src/components/LoginForm.vue

```diff
@@ -23,7 +55,46 @@ function loginUserAccount(e) {
 </script>
 
 <template>
-  <form @submit="loginUserAccount">
+  <form v-if="forgotPwd" @submit.prevent="forgetPassword">
+
+  <div v-if="successed" class="toast toast-top toast-right">
+    <div class="alert alert-success">
+      <span>Email sended !</span>
+    </div>
+  </div>
+  <div v-if="successed == false" class="toast toast-top toast-right">
+    <div class="alert alert-error">
+      <span>Error !</span>
+    </div>
+  </div>
+
+    <h3 class="text-3xl font-bold my-5">Forgot !</h3>
+    <span class="">Forgot your username or password ? </br>Recive and change it by mail :</span>
```

Comment:
`</br>` ca n'existe pas mais `<br />` oui
```suggestion
    <span>Forgot your username or password ? <br/>Recive and change it by mail :</span>
```